### ISSUE TITLE
Synergy breaks input in (some?) games (e.g. Terraria)

### ISSUE THREAD

**Issue by denilsonsa** [synergy/synergy#5147]
Related issue (marked as obsolete): #3129

My system: Windows 7 64-bit, Synergy v1.7.6-rc1-c97bd2c.

How to reproduce:

1. Leave Synergy service (server) running in background (bonus points if you forget it was even running). There is no need to connect any client, just the server is enough.
2. Try to launch Terraria game (I'm launching it [from Steam](http://store.steampowered.com/app/105600/)).
3. Case 1:
    1. Observe the mouse seems to work in menus.
    2. However, after interacting with the menus enough times (after a dozen or so clicks, or after a minute or so), mouse clicks stop working in the menus. Notice this happens even without actually starting a game world.
4. Case 2:
    1. After launching Terraria, start a new game using any character and any world.
    2. After the world loads, your character is at the very middle of the screen. By now, it has been a minute or so since Terraria was started.
    3. Try clicking with the mouse. For me, sometimes it worked, and then suddenly stopped working. And once it stopped working, it kept not-working forever.
    4. Try moving your character using WASD keys. Nothing happens.
    5. Try using the mouse wheel to select different weapon/item slots. Surprisingly, it works!
    6. Try opening the in-game menu. It may or may not work. Even if the menu opens, clicking may simply not work.
5. Go to Synergy and click on `Stop` to stop the background service.
6. The game controls start working immediately, no need to restart the game.

I've updated to 1.7.6-some-nightly-version because I wanted a working clipboard. But I remember that a previous version ~~(one of these: 1.4.17, 1.5.0, 1.6.3; I don't remember which one)~~ had no issues. That earlier version worked fine all the time.

**Comment by denilsonsa** [comment]
I've just downgraded to 1.6.3 and this issue is not present in that version. It seems I can play Terraria even with Synergy 1.6.3 running.

**Comment by denilsonsa** [comment]
I uninstalled synergy in order to install an older version. Then I tried installing versions 1.6.3, 1.7.1, 1.7.3, 1.7.4, 1.7.5, 1.7.6-rc1-c97bd2c, 1.7.6-rc2-a5acb08. All of them worked correctly. Sorry, but I can't reproduce this bug right now.

Yesterday, when the bug happened, I tried restarting the system and the bug remained. Today I've installed all of these versions without restarting. I'm not sure if that's related to the issue. I'll report back later, whenever I restart Windows.
